build/make/info.mk

```diff
@@ -1,5 +1,5 @@
 .PHONY: info
-info:
+info: ## Print build information
```

Comment:
This file has no caption or group description. If for any reason someone includes the makefiles in a different order the description would end up anywhere it does not belong.

I feel `info` does not necessarily must retain it's own file. It could also be moved to `variables.mk` without loss.